src/middlewared/middlewared/main.py

```diff
@@ -890,7 +880,7 @@ def __init__(
         self.__terminate_task = None
         self.jobs = JobsQueue(self)
         self.mocks = {}
-        self.socket_messages_queue = deque(maxlen=1000)
+        self.socket_messages_queue = deque(maxlen=50)
```

Comment:
How about 200? xD

Reply:
I've opened a random machine, started at Storage and went directly to Advanced -> Save Debug. There were 34 messages since clicking on Advanced menu item. Some of them were job related, but this is likely to happen on user's machine as well.
Since it's not unlikely for user to do something else before going to report an issue, I feel like 50 is going to be unusable and 200 is going to be barely usable.

Reply:
We are starting with 200. We can re-evaluate if you notice this is not enough after the release...